R/dust_class.R

```diff
@@ -124,6 +124,16 @@ dust_class <- R6::R6Class(
     reset = function(data, step) {
     },
 
+    ##' @description
+    ##' Set the 'data' element in a dust object while holding parameters,
```

Comment:
I think 'parameters' here is a little unclear, as doesn't data often refer to what we might call parameters? Does parameters mean number of particles, threads, index and state? And this doesn't change state, correct?

Reply:
parameters here should have been model state